src/app/hierarchical-grid/hierarchical-grid-row-pinning/hierarchical-grid-row-pinning.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, ViewChild } from "@angular/core";
+import { IgxHierarchicalGridComponent, IgxHierarchicalRowComponent,
+        IgxIconService, RowPinningPosition} from "igniteui-angular";
+import { IPinningConfig } from "igniteui-angular/lib/grids/common/grid.interface";
+import { icons } from "../../grid/services/svgIcons";
```

Comment:
Check the other samples that use this icon. They require a custom Stackblitz config that includes it in the bundle.